scss/detail.scss

```diff
@@ -3,9 +3,13 @@ $break-medium: 1024px;
 .book-container {
   background-color: $white;
   font-family: 'Roboto', sans-serif;
-  max-width: 900px;
-  padding: 24px;
+  max-width: 853px;
+  height: 570px;
+  padding: 32px 32px 56px 32px;
   width: 100%;
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: repeat(3, 1fr);
```

Comment:
Don't need to change this. In future projects you can also use grid areas. I think it's a more semantic approach to do this and easier to follow. Good work!! :rocket: